sanity/keylime-secure_mount/test.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+SECURE_DIR="/var/lib/keylime/secure"
+KEYLIME_UNIT_FILE="/usr/lib/systemd/system/keylime_agent.service"
```

Comment:
You should not work just with unit files in `/usr/lib/systemd/system`. Remember that our `setup/install-upstream..` tasks install the unit file into `/etc/systemd/system`. Also, `/etc/` has preference over `/usr/lib`. 
I am not sure what the "proper systemd way" of doing it would be. But at least you should check which of these two files exist, preferring the one in `/etc`.
Also, use rlFileBackup and rlFileRestore.